recirq/optimize/mpg.py

```diff
@@ -128,64 +110,60 @@ def value(self, t):
             the schedule value
         """
         m = t / self.decay_steps
-        if self.staircase == True:
+        if self.staircase:
             m = np.floor(m)
 
         return self.learning_rate * self.decay_rate ** m
 
 
-class Adam:
-    def __init__(
-        self, lr_schedule=ConstantSchedule(0.001), b1=0.9, b2=0.999, eps=10 ** -8
-    ):
-        """The optimizer Adam: a method for stochastic gradient descent  
+def Adam_update(
```

Comment:
I know the original authors style it as "Adam", but I think we should follow PEP8 here and call the function `adam_update`. 

An additional question: do we expect users to use this function or will it always be called from `model_policy_gradient`? That is -- can we make this private (by naming it `_adam_update`)?

Reply:
yes, that is right. I have fixed that.